packages/server/src/app/app.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const usersRouter = require('./models/user/user.controller')
+
+const app = express()
+
+app.disable('x-powered-by')
+
+app.use(express.json())
+app.use(express.urlencoded({ extended: false }))
+
+app.use('/api/user', usersRouter)
```

Comment:
I would suggest using api/v1/user because it gives us the ability to change the API without breaking the application from people using our API.